Control/AdaptivePurePursuit/AdaptiveLd.py

```
@@ -0,0 +1,388 @@
+            ),
+        }
+        # Publishers
+        self.subscribers = {
+            "Pose": rospy.Subscriber("/gazebo/model_states", ModelStates, self.updatePose),
+            "Path": rospy.Subscriber("/Path", Path, self.pathCallback),
+        }
+        # Subscribers
+
+        self.indexLD = 0
+        self.currentPosition = [0, 0, 0]  ##[x,y,theta]
+        DISTLD = 0.3
```

Ruff (0.8.2)

Local variable `DISTLD` is assigned to but never used

Remove assignment to unused variable `DISTLD`

(F841)